libs/ledger-live-common/src/hw/getBatteryStatus.test.ts

```diff
@@ -0,0 +1,152 @@
+import getBatteryStatus, { BatteryStatusTypes } from "./getBatteryStatus";
+import { ChargingModes } from "@ledgerhq/types-devices";
+
+const mockTransportGenerator = (out) => ({ send: () => out });
```

Comment:
Not in this PR: we should create a "mock builder" for our Transport, that would return by default a Transport with all its methods mocked with a default behavior, and with which we can also modify any of the Transport methods to a specific behavior

That would avoid the `@ts-ignore next-line` everywhere

Reply:
But i ❤️ the tests, so easy to read those different examples

Reply:
true, not for this pr though, https://ledgerhq.atlassian.net/browse/LIVE-4893 all yours :trollface: